src/screens/login/Login.jsx

```diff
@@ -83,7 +92,7 @@ const Login = ({ navigation }) => {
         <TouchableOpacity style={styles.forgot}>
           <Typography
             id="FORGOT_PASSWORD_QUESTION"
-            style={[styles.fontBasicText, { color: '#2C2C2C80' }]}
+            style={[globalStyles.basicText]}
```

Comment:
```suggestion
            style={[globalStyles.basicText]}
```
Same issue as above ^